tests/integration_tests/data/offline_workflow_config/core/core_api.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2018 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+#
+
+location:
+    type: app_store
+    name: tk-core
+    version: v0.18.133
```

Comment:
does this mean that we always run the test against this core? A comment here would be helpful! We should run it against the core in this repo, right?

Reply:
Great question, I thought about it.

This test's goal is to ensure that we can bootstrap into a config that has a cached copy of tk-core inside it's `bundle_cache` folder. So
- it's not important which core we will bootstrap into, it can be any core
- it needs to be cachable and a path descriptor can't be